MetamodelAlgemeen.md

```diff
@@ -766,9 +766,9 @@ Een *Keuze* is een opsomming van meerdere modelelementen, waarbij er maar van é
 
 *Toelichting:* Er kan altijd maar één van de mogelijkheden gekozen worden. De keuze is voor een aantal use cases een alternatieve manier voor het modelleren van een constraint.
```

Comment:
Voeg toe: Een keuze als modelleerconstruct heeft o.a. als voordeel dat het model driven verwerkt kan worden in schemaconstructies.